client/src/pages/customer-pages/FlightInfo/index.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from "react";
+
+import { CustomerPageNavBar, Tabs, Card } from "../../../components";
+import "./flightinfo.css";
+
+class FlightRoute extends Component {
+  render() {
+    return (
+      <div className="flightroute-container">
+        <CustomerPageNavBar />
+        <Tabs classNameMap="map-style" />
+        <div className="card-container">
+          <Card
+            cardTitle="Origin"
+            info={[
```

Comment:
put this array in a variable/state/props and pass it to the component, do not stack js in your jsx.

Reply:
Yeah we should do this, but for now, we will finish the admin panel to and later we will start working on customer views, This is balsam work and she made this pull request the last minute before she leave. it wasn't supposed to be in a pull request.